website/docs/r/filter.html.markdown

```diff
@@ -13,40 +13,37 @@ Manages ACI Filter
 ## Example Usage ##
 
 ```hcl
-	resource "aci_filter" "foofilter" {
-		tenant_dn   = aci_tenant.dev_tenant.id
-		description = "From Terraform"
-		name        = "demo_filter"
-		annotation  = "tag_filter"
-		name_alias  = "alias_filter"
-	}
+
+resource "aci_filter" "example" {
+  tenant_dn   = aci_tenant.dev_tenant.id
+  description = "From Terraform"
+  name        = "demo_filter"
+  annotation  = "tag_filter"
+  name_alias  = "alias_filter"
+}
+
 ```
+
 ## Argument Reference ##
 * `tenant_dn` - (Required) Distinguished name of parent Tenant object.
 * `name` - (Required) Name of Object filter.
 * `description` - (Optional) Description for object filter.
 * `annotation` - (Optional) Annotation for object filter.
 * `name_alias` - (Optional) Name alias for object filter.
-
-* `relation_vz_rs_filt_graph_att` - (Optional) Relation to class vnsInTerm. Cardinality - N_TO_ONE. Type - String.
-                
-* `relation_vz_rs_fwd_r_flt_p_att` - (Optional) Relation to class vzAFilterableUnit. Cardinality - N_TO_ONE. Type - String.
+* `relation_vz_rs_filt_graph_att` - (Optional) Relation to class vnsInTerm. Type: String.
                 
-* `relation_vz_rs_rev_r_flt_p_att` - (Optional) Relation to class vzAFilterableUnit. Cardinality - N_TO_ONE. Type - String.
-                
-
+* `relation_vz_rs_fwd_r_flt_p_att` - (Optional) **Deprecated** Relation to class vzRsFwdRFltPAtt. Type: String.      
+* `relation_vz_rs_rev_r_flt_p_att` - (Optional) **Deprecated** Relation to class vzRsRevRFltPAtt. Type: String.
 
 ## Attribute Reference
 
-The only attribute that this resource exports is the `id`, which is set to the
-Dn of the Filter.
+The only attribute that this resource exports is the `id`, which is set to the Dn of the Filter.
 
 ## Importing ##
 
 An existing Filter can be [imported][docs-import] into this resource via its Dn, via the following command:
 [docs-import]: https://www.terraform.io/docs/import/index.html
 
-
 ```
 terraform import aci_filter.example <Dn>
-```
\ No newline at end of file
+```
```

Comment:
Shouldn't we add the new import description just as follow?

Starting in Terraform version 1.5, an existing object filter can be imported 
using [import blocks](https://developer.hashicorp.com/terraform/language/import) via the following configuration:

```
import {
  id = "<Dn>"
  to = aci_filter.example
}
```